optimize.Dockerfile

```diff
@@ -75,6 +75,8 @@ VOLUME ${OPTIMIZE_HOME}/logs
 
 RUN addgroup --gid 1001 camunda && \
     adduser -D -h ${OPTIMIZE_HOME} -G camunda -u 1001 camunda && \
+    apk add --no-cache bash curl tini openjdk21-jre tzdata && \
+    apk -U upgrade && \
```

Comment:
🔧  is that actually related/needed or just an additional change to update all packages?